OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -63,6 +63,8 @@ public class ArmamentInfo : ConditionalTraitInfo, Requires<AttackBaseInfo>
 		[Desc("Use multiple muzzle images if non-zero")]
 		public readonly int MuzzleSplitFacings = 0;
 
+		public readonly bool ShootAtClosestTargetablePosition = false;
```

Comment:
> I'm in favor of b).

Me too.

Ping @SoScared, @AoAGeneral who can spread the word around the others who are interested in balance.